utils/api.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const ncNewsAPI = axios.create({
+  baseURL: "https://nc-news-dwainfco.onrender.com/api",
+});
+
+export function getAllArticles() {
+    return ncNewsAPI.get("/articles", {
```

Comment:
It would make your components code a bit cleaner if you took the articles off the relevant keys here rather than in the component 🔍